src/ion/data_structures.rs

```diff
@@ -216,14 +216,12 @@ impl LiveBundle {
         minimal: bool,
         fixed: bool,
         fixed_def: bool,
-        stack: bool,
     ) {
         debug_assert!(spill_weight <= BUNDLE_MAX_SPILL_WEIGHT);
         self.spill_weight_and_props = spill_weight
             | (if minimal { 1 << 31 } else { 0 })
             | (if fixed { 1 << 30 } else { 0 })
-            | (if fixed_def { 1 << 29 } else { 0 })
-            | (if stack { 1 << 28 } else { 0 });
+            | (if fixed_def { 1 << 29 } else { 0 });
```

Comment:
Since we get bit 28 back here can we update `BUNDLE_MAX_SPILL_WEIGHT` as well (so the saturation point moves a little further out, leading to maybe slightly better alloc in deeply nested loops)?

Reply:
Great idea!